docs/methodology.md

```diff
@@ -1,48 +1,56 @@
 # Methodology
-Below we describe the methodology used to benchmark [CLP] and other tools.
+
+We benchmark ingestion and query performance for each tool. Since [CLP] supports both 
+**unstructured** and **dynamically-structured** logs, so we have separate benchmarks for each. 
+Below, we describe the datasets, benchmark machines, execution flow, and metrics collected.
 
 ## Test datasets
+
 There are two types of logs used in the benchmark:
-+ **Unstructured logs** are log events that don't follow a predefined format or structure, making it
+
+- **Unstructured logs** are log events that don't follow a predefined format or structure, making it
   difficult to automatically parse or analyze them. Unstructured logs are often just text
-  interspersed with variable values like the log event's timestamp, log-level, and others such as
+  interspersed with variable values like the log event's timestamp, log level, and others such as
   usernames, etc. For example:
+
   ```
   2018-06-05 06:15:29,701 DEBUG org.apache.hadoop.util.Shell: setsid exited with exit code 0
   ```
 
-+ **Semi-structured logs** do have follow a predefined format, making them parseable, but their
-  schema is not rigid. Semi-structured log events usually have a few consistent fields (e.g., the
+- **Dynamically-structured logs** do have follow a predefined format, making them parseable, but 
+  their
+  schema is not rigid. Dynamically-structured log events usually have a few consistent fields (e.
+  g., the
   event's timestamp and log level) but the rest may appear and disappear across each event. For
   example:
 
   ```json
   {
```

Comment:
Based on the description, this example is a little confusing. Can we:

* Shorten it to just a few fields that can be displayed on one line.
* Add another record to show that some fields always exist and some fields are added or removed between records.